### DISCUSSION TITLE
unknown flag when passing arguments from CLI to Task

### DISCUSSION THREAD

**Discussion by m-soliman3** [go-task/task#514] [Q&A]
I am trying to use the pretty new feature to pass arguments from CLI when running task to the command itself.

1 add `{{.CLI_ARGS}}` at the end of the task `cmds`
2 used the command as` task myTask -- --myArgumentToPass`

I always get an error that `--myArgumentToPass` is unknown flag. Task doesn't understand that I am not sending it aan argument and want it only to pass it to my task itself. Does anyone know how to solve this?

**Comment by andreynering** [comment]
Hi @m-saeed-soliman,

Can you mention which OS, shell and Task version you are using?

Someone have said that on Powershell on Windows you have to quote `--` to make it work.

**Comment by m-soliman3** [comment]
Thank you for reply.
It worked. I jau had to not use arrow function and use normal function
